frontend/src/views/applications/ApplicationsHistoryView.vue

```diff
@@ -21,7 +21,10 @@
             </v-card-title>
             <v-card-text class="text-center d-flex flex-column align-center pt-4 pb-0">
               <div v-if="isAddCoreApplicationAllowed">Before starting an application, verify your organization and facility details in Account Management.</div>
-              <div v-else>The Application intake is currently closed.</div>
+              <div v-else>
+                <div v-if="!isCCOFEnrolmentCheckSatisfied">Enrolment in CCOF for greater than 1 year is required to apply.</div>
```

Comment:
I might be tempted to use either a ternary expression with {{}} or a computed method to determine the text. It will avoid the extra level of nesting and unnecessary divs.